packages/suite/src/support/messages.ts

```diff
@@ -9228,6 +9240,10 @@ export default defineMessages({
         id: 'TR_VIEW_ONLY',
         defaultMessage: 'View-only',
     },
+    TR_URL_IN_TOKEN: {
+        id: 'TR_URL_IN_TOKEN',
+        defaultMessage: 'Never visit URLs provided in tokens. They are usually scams.',
```

Comment:
What do you mean? The copy is grabbed from Figma.

Reply:
![Screenshot 2024-06-06 at 13 49 40](https://github.com/trezor/trezor-suite/assets/33235762/a6aaa6d6-07bd-4b5d-a5e2-4ee34ae6c9fd)

I dont think so 😀 

Nevermind I think it would be better to use `Never visit URLs in token names and symbols. They are usually scams.`

Reply:
Oh so I putted there old copy that was there. `Change it to Never visit URLs in token names or symbols. They are usually scams.`